react-main/src/app/components/Game/index.js

```diff
@@ -14,9 +11,9 @@ class Game extends Component {
     this.props.square(i)
   );
 
-  handleHistoryClick = step => {
-    return this.props.historySelected(step);
-  };
+  handleHistoryClick = step => (
+    this.props.historySelected(step)
```

Comment:
Try to keet the function in the same line if we can:

`handleHistoryClick = step => this.props.historySelected(step)`